### ISSUE TITLE
Active search bar glow

### ISSUE THREAD

**Issue by lzrscg** [modularcloud/explorer#64]
# Description
According to the Figma designs, when the search bar is active, it should have a glow effect.

# Screenshot / Video

# URL / Steps to reproduce

# Expected behavior

# Notes

**Comment by Aswanth-c** [comment]
I think the given css is wrong. When I try to implement it.
![image](https://github.com/modularcloud/explorer/assets/57211637/14f033fb-5384-4e45-947a-10cd4f58e6bf)
It displays like this 
can I get access to the figma file where I can check this?

**Comment by lzrscg** [comment]
@mateuszsinilo can you double check this

here is the design for your reference @Aswanth-c 
https://www.figma.com/file/JwErDZu5OhBpzSKgLRhDPw/Client-%2F-Celestia-Scan?type=design&node-id=1123%3A17428&t=eNrGFQ9R91wKC12y-1

**Comment by mateuszsinilo** [comment]
> @mateuszsinilo can you double check this
> 
> here is the design for your reference @Aswanth-c https://www.figma.com/file/JwErDZu5OhBpzSKgLRhDPw/Client-%2F-Celestia-Scan?type=design&node-id=1123%3A17428&t=eNrGFQ9R91wKC12y-1

yes, I double checked it, and it's right. But I know where the problem is. James used many of diferrent styles to achieve this effect. So for now, I recommend to use a one color for border, and here's a hex '#9DAAE2' for that color. And there was used the blur behind the searchbar, so in original design, it's like two search bars - one below with blur(5px) and one above with the gradient-stroke. I don't know if it makes sense ;)

**Comment by Aswanth-c** [comment]
> > @mateuszsinilo can you double check this
> > here is the design for your reference @Aswanth-c https://www.figma.com/file/JwErDZu5OhBpzSKgLRhDPw/Client-%2F-Celestia-Scan?type=design&node-id=1123%3A17428&t=eNrGFQ9R91wKC12y-1
> 
> yes, I double checked it, and it's right. But I know where the problem is. James used many of diferrent styles to achieve this effect. So for now, I recommend to use a one color for border, and here's a hex '#9DAAE2' for that color. And there was used the blur behind the searchbar, so in original design, it's like two search bars - one below with blur(5px) and one above with the gradient-stroke. I don't know if it makes sense ;)

Gotcha I used the single Color hex value for border  And created the  pull request
https://github.com/modularcloud/explorer/pull/89
